src/settings/toml/manifest.rs

```diff
@@ -63,6 +63,8 @@ pub struct Manifest {
     pub durable_objects: Option<DurableObjects>,
     #[serde(default, with = "string_empty_as_none")]
     pub usage_model: Option<UsageModel>,
+    pub compatibility_date: Option<String>,
+    pub compatibility_flags: Option<Vec<String>>,
```

Comment:
nit: `Option<Vec>` has two ways to represent "no flags": `None` and `Some(vec![])`. Can you change this to be Vec directly so there's only one way and callers don't have to check if the vec is empty?
```suggestion
    #[serde(default)]
    pub compatibility_flags: Vec<String>,
```

Reply:
Actually, you do use `Vec` below for `Manifest`, so this was probably just because you didn't see `serde(default)`.

Reply:
Yeah I don't know anything about serde, so I had no idea I could use `#[serde(default)]`.